Gopkg.toml

```diff
@@ -9,7 +9,8 @@ required = [
 
 [[override]]
   name = "github.com/gardener/gardener"
-  revision = "7798c480cfd8cb9f7c6bfd2e4fd7441a7a469db7"
+  source = "github.com/swapnilgm/gardener"
```

Comment:
In order to get this PR merged, can you please find a separate one to `gardener/gardener` that only includes the changes that are needed for this PR?

Reply:
So, basically you want me to split PR gardener/gardener#1128. Will do it.

Reply:
Actually both the PR, this and  gardener/gardener#1128 should go hand in hand with next release on gardener-extensions and gardener, so isn't it better to have it in gardener/gardener#1128. So, once it gets the LGTM, while merging it i'll update the `gopkg.toml`. By any chance the review/merging on one or other PR gets delayed, maybe  if you have plan to make next release on one of them,  before another one, it will be messed up. Just being little paranoidal about it. WDYS?

Reply:
The PR's don't matter, the actual releases do. `gardener/gardener-extensions` can have PRs merged already while `gardener/gardener` does not. In the end the versions that are deployed together must match.

So, I rather like to get this one in, so I guess it makes sense to split gardener/gardener#1128 into two parts such that this PR can get merged. :)

Reply:
Cool no arguments. I'll split it.

Reply:
done.